### DISCUSSION TITLE
Remove context menu transitions

### DISCUSSION THREAD

**Discussion by krupakov** [quasarframework/quasar#13271] [General - Components / Directives / etc] [answered]
Hello, I'm currently using desktop-like menu from this page
[https://quasar.dev/vue-components/menu#submenus](url)

Is there a way to remove all context menu transitions?

![menu](https://user-images.githubusercontent.com/58831355/165587372-5b627508-5acb-4a03-bea2-d107280a9a65.png)

**Comment by MilosPaunovic** [comment, marked as answer]
The easiest way would be to pass zero as a value of transition duration parameter (`:transition-duration="0"`)
[https://codepen.io/milospaunovic/pen/WNdVaba?editors=1010](https://codepen.io/milospaunovic/pen/WNdVaba?editors=1010)

**Reply by krupakov** [reply]
Thanks a lot!
